lib/src/tools/analyze_tool.dart

```diff
@@ -52,25 +53,33 @@ class AnalyzeTool extends DevTool {
   /// files in the current working directory.
   List<Glob> include;
 
+  /// The default tool for analysis will be `dartanalyzer` unless opted in here
+  /// to utilize `dart analyze`.
```

Comment:
I think we can avoid needing to document this explicitly (other than in the changelog) since the package has its SDK minimum set to a version that safely ensures these commands are available:
https://github.com/Workiva/dart_dev/blob/master/pubspec.yaml#L7